static/js/reducers/courses.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import { GET, INITIAL_STATE } from "redux-hammock/constants"
+import * as api from "../lib/api/api"
+
+export const courseFacetsEndpoint = {
+  name:              "coursefacets",
+  verbs:             [GET],
+  initialState:      { ...INITIAL_STATE, data: new Map() },
+  getFunc:           () => api.aggregate(["topics", "platform"]),
+  getSuccessHandler: (response: Object): any => {
```

Comment:
It doesn't look like this or `getFunc` is covered by tests. Can you investigate?

Reply:
Done